website/source/docs/partnerships/index.html.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: "docs"
+page_title: "Consul Integration Program"
+sidebar_current: "docs-consul-program"
+description: |-
+  Guide to partnership integrations for Consul.
+---
+
+# Consul Integration Program
+
+The HashiCorp Consul Integration Program enables vendors to build integrations with HashiCorp Consul that are tested and approved by HashiCorp. The program is intended to be largely self-service with links to resources, code samples, documentation, and clear integration steps.
+
+## Types of Consul Integrations
+
+By leveraging Consul’s RESTful HTTP API system, vendors are able to build extensible integrations at the data plane, platform, and the infrastructure layer to extend Consul’s functionalities. These integrations can be performed with the OSS (open source) version of Consul. Integrations with advanced network segmentation, advanced federation, and advanced read scalability need to be tested against Consul Enterprise since these features are only supported by Consul Enterprise.
+
+<div class="center">
+[![Consul Architecture](/assets/images/consul_ecosystem_diagram.png)](/assets/images/consul_ecosystem_diagram.png)
+</div>
+
+**Data Plane**: These integrations automate IP updates of load balancers by leveraging Consul service discovery,  automate firewall security policy updates by leveraging Consul intentions within a centralized management tool, extend sidecar proxies to support Consul connect, and extend API gateways to allow Consul to route incoming traffic to the proxies for Connect-enabled services.
+
+**Platform**: Consul has a client-server architecture and is the control plane for the service mesh. No integrations at this layer.
+
+**Platform**: These integrations leverage automation of Consul agent deployment, configuration, and management through cloud and PaaS provisioning and orchestration tools such as Kubernetes and Pivotal Cloud Foundry (PCF). They include the Consul agent running in both client and server mode.
+
+**Infrastructure**: These integrations extend Consul’s connect certificate management, secure ACL configuration, observability metrics and logging, and service discovery that allows for dynamic service mapping with ITSM tools.
```

Comment:
Remove "connect", since Consul CA is now used for other purposes as well (e.g. autoTLS).

Reply:
Thanks! fixed!